source/Nuke.Common/Tools/DotNet/DotNet.json

```diff
@@ -1081,6 +1081,64 @@
           }
         ]
       }
+    },
+    {
+      "help": "The <c>dotnet workload install</c> command installs one or more optional workloads. Optional workloads can be installed on top of the .NET SDK to provide support for various application types, such as .NET MAUI and Blazor WebAssembly AOT.",
+      "postfix": "WorkloadInstall",
+      "omitCommonProperties": true,
+      "definiteArgument": "workload install",
+      "settingsClass": {
+        "properties": [
+          {
+            "name": "WorkloadId",
+            "type": "List<string>",
+            "format": "{value} ",
+            "help": "The workload ID or multiple IDs to install."
```

Comment:
I wasn't sure here how to do this correct.. My first thought was to append a space after the value. Is that correct?